src/penn_chime/models.py

```diff
@@ -340,15 +345,3 @@ def build_census_df(
             for key, los in lengths_of_stay.items()
         }
     })
-
-
-
-#############
-##  UTILS  ##
-#############
```

Comment:
are we reverting back to before https://github.com/CodeForPhilly/chime/issues/90 ?

Reply:
no i think i see `def get_growth` or something that makes sense.

Reply:
There we two growth functions, one returned a percent and the other returned a rate. Consolidated these.